src/app/work-item/work-item-detail/work-item-comment/work-item-comment.component.html

```diff
@@ -20,7 +20,7 @@
               data-placeholder="Add a new comment..." #comment></p>
       </div>
       <div class="flex-container flex-grow-1 in-column-direction comments-outer-wrapper">
-        <div *ngFor='let comment of workItem.relationships?.comments?.data?.slice(); let counter = index' class="comments-wrap">
+        <div *ngFor='let comment of comments.slice(); let counter = index' class="comments-wrap">
```

Comment:
Unsure, probably to not alter the original array.
Left it as it is, just to avoid any unforeseen regression.

Should I remove it?